cli/testdata/deploy/main.go

```diff
@@ -13,7 +13,7 @@ var key = "key"
 
 const mgmtKey = "mgmt"
 
-func _deploy(isUpdate bool) {
+func _deploy(data interface{}, isUpdate bool) {
```

Comment:
We need to change all our examples (`neo-go/examples`) because some of them also have `deploy` method.

Reply:
Fixed.